docs/reference/mapping/params/fielddata.asciidoc

```diff
@@ -1,134 +0,0 @@
-[[fielddata]]
-=== `fielddata`
```

Comment:
It's best practice to add redirects for any removed fields. I added a redirect for this page with 391ab35414f190530d240a554831b17b48ac0803 and c121d75cee297a5cabf41e657e1c740c67287b92